boot/bootutil/include/bootutil/image.h

```diff
@@ -39,13 +39,17 @@ struct flash_area;
  * Image header flags.
  */
 #define IMAGE_F_PIC                      0x00000001 /* Not supported. */
-#define IMAGE_F_NON_BOOTABLE             0x00000010 /* Split image app. */
+#define IMAGE_F_NON_BOOTABLE             0x00000002 /* Split image app. */
+/*
+ * FIXME: This image was encrypted...
+ */
+#define IMAGE_F_ENCRYPTED                0x00000004
 /*
  * Indicates that this image should be loaded into RAM instead of run
  * directly from flash.  The address to load should be in the
  * ih_load_addr field of the header.
  */
-#define IMAGE_F_RAM_LOAD                 0x00000020
+#define IMAGE_F_RAM_LOAD                 0x00000020 //FIXME
```

Comment:
Why the comment?

Reply:
Yup, unfortunately I can't remember the exact issue, but I think I was trying to find out the reason for the gap in the bitmask, and while looking for other places that define this (simulator, imgtool, newt and v1 header in mynewt) the values didn't match!

So newt for v2 format has (ENCRYPTED on my PR only!):

```
IMAGE_F_PIC          = 0x00000001
IMAGE_F_NON_BOOTABLE = 0x00000002 /* non bootable image */
IMAGE_F_ENCRYPTED    = 0x00000004 /* encrypted image */
```

imgtool has:

```
IMAGE_F = {
        'PIC':                   0x0000001,
        'NON_BOOTABLE':          0x0000010,
        'ENCRYPTED':             0x0000004,
}
```

Here already `NON_BOOTABLE` does not match.

The simulator has:

```
pub enum TlvFlags {
    PIC = 0x01,
    NON_BOOTABLE = 0x02,
    ENCRYPTED = 0x04,
    RAM_LOAD = 0x20,
}
```
And mcuboot itself:

```
#define IMAGE_F_PIC                      0x00000001 /* Not supported. */
#define IMAGE_F_NON_BOOTABLE             0x00000002 /* Split image app. */
#define IMAGE_F_ENCRYPTED                0x00000004
#define IMAGE_F_RAM_LOAD                 0x00000020
```
In the end I think the policy was to try not to reuse defines that clash with those from v1, but maybe we could have just started with new values, since the magic already differentiates the header versions, and instead of jumping over bits just increase them in the usual sequential way. I will remove this now when squashing the commits to cleanup and open an issue to discuss it.